students/david/src/01-fibonacci/index.ts

```diff
@@ -0,0 +1,17 @@
+import iterativeFibonacci from "./iterativeFibonacci";
+import recursiveFibonacci from "./recursiveFibonacci";
+
+let numberOfIterations:number;
```

Comment:
You could merge declaration and assign of those variables and change it to const instead of let